src/errors/checkXcmTxInputs.ts

```diff
@@ -544,16 +544,16 @@ export const checkIsValidSystemChainAssetId = async (
  * @param specName
  * @param relayChainInfo
  */
-const checkParaToSystemAssetId = async (
+const checkParaToAssetHubAssetId = async (
 	api: ApiPromise,
 	assetId: string,
 	relayChainInfo: ChainInfo,
 	registry: Registry,
 	isForeignAssetsTransfer: boolean
 ) => {
-	const systemParachainId = SYSTEM_PARACHAINS_IDS[0];
-	const systemParachainInfo = relayChainInfo[systemParachainId];
-	const systemSpecName = systemParachainInfo.specName;
+	const assetHubChainId = ASSET_HUB_CHAIN_ID;
```

Comment:
Little nit, but I think we can remove this and just do `relayChainInfo[ASSET_HUB_CHAIN_ID];`